README.md

```diff
@@ -7,7 +7,10 @@ an index query mechanism for datasets provided by [Common Crawl](https://commonc
 ## Usage & Installation
 To run locally, please install with `pip install -r requirements.txt`
 
-Common Crawl stores data on Amazon S3 and the index is publicly accessible from S3.
+Common Crawl stores data on Amazon S3 and the index is available for authenticated users from AWS S3. Set the following environment variables to access the index:
+
```

Comment:
Strictly speaking, the index is available for all users, even without authentication
- only cluster.idx and metadata.yaml are required to download, either via `aws s3 sync` or by fetching via http://data.commoncrawl.org/
- cdx-*.gz files are read from S3 - the access method is configured in config.yaml

There are many more methods to [configure AWS credentials](https://docs.aws.amazon.com/accounts/latest/reference/credentials-access-keys-best-practices.html). Personally, I'd avoid to set credentials via environment variables because it's possible that the credentials leak into log files.

Reply:
I thought all files needed authentication. Thanks for clarification.